cli/src/semgrep/commands/ci.py

```diff
@@ -367,6 +368,8 @@ def ci(
         scan_handler and "secrets" in scan_handler.enabled_products
     )
 
+    disable_secrets_validation = disable_secrets_validation_flag
```

Comment:
Could we just skip this local? Not seeing why this would be better than just using the argument later instead.